changelogs/current.yaml

```diff
@@ -31,6 +31,11 @@ minor_behavior_changes:
   change: |
     Enable QUICHE request and response headers validation. This behavior can be reverted by setting runtime flag
     ``envoy.reloadable_features.FLAGS_envoy_quic_reloadable_flag_quic_act_upon_invalid_header`` to false.
+- area: zone-aware routing
+  change: |
+    Zone-aware routing is now enabled even when the originating and upstream cluster have different numbers of zones.
+    This behavioral change can be reverted by setting runtime guard
+    ``envoy.reloadable_features.enable_zone_routing_different_zone_counts`` to false.
```

Comment:
I would call this a major behavior change. Can you move this to the new features section just to make it more clear that this is changing?

Reply:
I've refactored and combined the changelog entries and moved it into the major behavior change section

lmk how this looks or if I can improve this further